src/qml/NightstandWatchfacePage.qml

```diff
@@ -0,0 +1,221 @@
+                    radius: DeviceInfo.hasRoundScreen ?
+                                width :
+                                Dims.w(3)
+                    clip: true
+
+                    Image {
+                        id: previewPng
+
+                        property string previewImg: (assetPath + "watchfaces-preview/" + Dims.w(40) + "/" + fileName).slice(0, -4) + ".png"
+                        property bool previewExists: FileInfo.exists(previewImg)
+
+                        z: 1
+                        anchors.centerIn: parent
+                        width: Math.min(parent.width, parent.height)
+                        height: width
+                        source: !previewExists ? "" : previewImg
+                        asynchronous: true
+                    }
+
+                    Loader {
+                        id: previewQml
+
+                        z: 2
+                        visible: !previewPng.previewExists
+                        active: visible
+                        anchors.centerIn: parent
+                        width: Math.min(parent.width, parent.height)
+                        height: width
+                        source: folderModel.folder + "/" + fileName
+                        asynchronous: true
+                    }
+
+                    MouseArea {
+                        anchors.fill: parent
+                        onClicked: watchfaceNightstandSource.value = folderModel.folder + "/" + fileName
+                    }
+
+                    Image {
+                        id: wallpaperBack
+
+                        property string previewSizePath: "wallpapers/" + Dims.w(50)
+                        property string wallpaperPreviewImg: wallpaperSource.value.replace("\wallpapers/full/g", previewSizePath).slice(0, -3) + "jpg"
+
+                        z: 0
+                        anchors.fill: parent
+                        fillMode: Image.PreserveAspectFit
+                        visible: opacity
+                        opacity: watchfaceNightstandSource.value === folderModel.folder + "/" + fileName ? 1 : 0
+                        source: opacity > 0 ? FileInfo.exists(wallpaperPreviewImg) ?
+                                    wallpaperPreviewImg :
+                                    wallpaperSource.value : ""
+                        Behavior on opacity { NumberAnimation { duration: 100 } }
+                    }
+
+                    layer.enabled: true
+                    layer.effect: OpacityMask {
+                        maskSource:
+                            Rectangle {
+                                anchors.centerIn: parent
+                                width: Math.min(wallpaperBack.width, wallpaperBack.height)
+                                height: width
+                                radius: maskArea.radius
+                            }
+                    }
+                }
+
+                Icon {
+                    name: "ios-moon-outline"
```

Comment:
I don't think this moon icon helps now that we have split the watchfaces selector in two. Back when it was the same page, it underlined the difference between the "selected" watchface and the nighstand watchface but now this is just a confusing way to show which is the nighstand watchface. Let's use the same icon everywhere for consistency, user should know from the context

Reply:
Done!